api/client/service/opts.go

```diff
@@ -528,6 +531,7 @@ const (
 	flagEnv                  = "env"
 	flagEnvRemove            = "env-rm"
 	flagEnvAdd               = "env-add"
+	flagGroupAdd             = "group-add"
```

Comment:
@cpuguy83 This PR is about `service create` and `--group-add` is referring to adding secondary groups (in addition to the one specified by `--user user:group`).

I think the `group-rm` might fit `service update --group-rm ` case. I can work on the PR for `service update` if this is expected.

cc @thaJeztah